### PR TITLE
Add mailing command

### PR DESCRIPTION
**Pull request by Isengo1989**
As talked about with @shyim in Slack here is a simple integration with email instead of webhooks.

The integration with the notifier component from symfony is nice as well, but uses the message queue to send messages. There is always the possibility of adding an own webhook interface, but probably just overkill atm (as well as adding an own mailtemplate/migration etc. to the mail integration).

Feel free to mention the other dev that you referred to.

**Usage**

3 configs are added

- Mail
- Grace time for queue (those are curerntly fixed in the code of the healthchecker)
- Grace time for task (those are curerntly fixed in the code of the healthchecker)

**First argument must be a Saleschannel ID**

`bin/console frosh:monitor 4606300361c34ae3b69d813bb292f823`

**Email can be overwritten in command:**

bin/console frosh:monitor 4606300361c34ae3b69d813bb292f823 --email=michahobert@test.de

**PS**: Small fixes done by PHPCSFIXER

### CODE REVIEW

File: src/Components/Elasticsearch/EsProductDefinition.php [hunk 1]
```diff
@@ -239,7 +238,7 @@ private function buildCoalesce(array $fields, Context $context): string
             }
         }
 
-        return substr($coalesce, 0, -1) . ')';
+        return mb_substr($coalesce, 0, -1) . ')';
```
Comment (shyim) [inline src/Components/Elasticsearch/EsProductDefinition.php:241]: why this change?

File: src/Resources/config/config.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/master/src/Core/System/SystemConfig/Schema/config.xsd">
+
+    <card>
+        <title>Monitoring</title>
+        <title lang="de-DE">Überwachung</title>
+
+        <input-field type="text">
```
Comment (shyim) [inline src/Resources/config/config.xml:9]: do you maybe want this in the symfony config? we have already one

File: src/Components/Elasticsearch/EsProductDefinition.php (hunk 1 above)
Comment (shyim) [reply src/Components/Elasticsearch/EsProductDefinition.php:241]: we should disable that 😂

File: src/Resources/config/config.xml (hunk 2 above)
Comment (Isengo1989) [reply src/Resources/config/config.xml:9]: You mean in the SymfonyConfigCompilerPass ? What would be the benefit of passing it there?

Comment (shyim) [reply src/Resources/config/config.xml:9]: you can configure it using config/packages/frosh_tools.yaml. idk if you want that a shop owner can change this

File: src/Components/Elasticsearch/EsProductDefinition.php (hunk 1 above)
Comment (Isengo1989) [reply src/Components/Elasticsearch/EsProductDefinition.php:241]: As far as I saw you used it in other classes as well, keep it similar I would say. Only diff is apperently that it will handle UTF-8 encoding better. You can disable it in the php-cs-fixer file tho, i am emotionless in that topic :laughing:

File: src/Resources/config/config.xml (hunk 2 above)
Comment (Isengo1989) [reply src/Resources/config/config.xml:9]: Hmm I would say so, keep the obstacle small for a simple monitoring like that.

This config/packages/frosh_tools.yaml would be great in the Readme :slightly_smiling_face:

**Comment by Isengo1989** [comment]
@shyim what is missing here? Should I revert the mb_substr and what is your / the rest's decision on the plugin config?
